README.md

```diff
@@ -31,7 +31,35 @@ let sig = c.signAlgoTransaction(context: KeyContext.Address, account: 0, change:
 Where prefixEncodedTx is a transaction that has been compiled with the SDK's transaction builder. The signature returned can be verified against the public key:
 
 ```swift
-c.verifyWithPublicKey(signature: sig, message: prefixEncodedTx, publicKey: pk)
+let pk = c.keyGen(context: KeyContext.Address, account: 0, change: 0, keyIndex: 0)
+let result = c.verifyWithPublicKey(signature: sig, message: prefixEncodedTx, publicKey: pk)
+```
+
+It is also possible to sign arbitrary data. You need to specify a JSON scehma and encoding type (none, base64, msgpack).
+
+For example (with schema under "schemas/auth.request.json"):
+
+```swift
+let schema = try JSONSerialization.jsonObject(with: try Data(contentsOf: URL(fileURLWithPath: #file).deletingLastPathComponent().appendingPathComponent("schemas/auth.request.json")), options: []) as! [String: Any]
```

Comment:
is it possible to format this line and add some explanations?

Reply:
Thanks for pointing this out, it will now use Schema